src/analysisd/analysisd.c

```diff
@@ -2394,3 +2546,156 @@ void w_init_queues(){
     /* Initialize upgrade module message queue */
     upgrade_module_input = queue_init(getDefine_Int("analysisd", "upgrade_queue_size", 128, 2000000));
 }
+
+static void load_limits(void) {
+
+    cJSON * analysisd_limits = NULL;
+    int err = load_limits_file("wazuh-analysisd", &analysisd_limits);
+
+    if (err == LIMITS_SUCCESS && analysisd_limits) {
+
+        unsigned int old_timeframe = limits.timeframe;
+        cJSON *timeframe_eps;
+
+        w_mutex_lock(&limit_eps_mutex);
```

Comment:
I think we should move this lock to the line https://github.com/wazuh/wazuh/pull/13398/files#diff-54a0031895d30cd0b1e2b1589ca78822351f02ac6d1d5fe5f3e63726319df685R2554
and release it at the end of the function.

Reply:
Done